vocab-builder-cli/src/main/java/com/github/tkurz/sesame/vocab/Main.java

```diff
@@ -258,10 +258,10 @@ private static Options getCliOpts() {
                 .create('c'));
 
         o.addOption(OptionBuilder
-                .withLongOpt("stringConstantPrefix")
```

Comment:
To avoid collisions you can set the suffix. The prefix would always be `""`.

But of course I can add an option to also set the suffix...